retrieval/discovery/marathon/url.go

```diff
@@ -0,0 +1,14 @@
+package marathon
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+const appsURLFormat string = "http://%s/v2/apps/?embed=apps.tasks"
+
+// RandomAppsURL randomly selects a server from an array and creates an URL pointing to the app list
+func RandomAppsURL(servers []string) string {
```

Comment:
People not concerned with the Prometheus setup might make incremental changes to the infrastructure which effectively moves all nodes. In an instant the setup is broken.

Not saying it's a pressuring issue but if it can be avoided in the future, it should.